EasyPost.Tests.Net/OrderTest.cs

```diff
@@ -1,211 +1,85 @@
-        //         to_address = Address.Create(toAddress),
-        //         from_address = Address.Create(fromAddress),
-        //         reference = "OrderRef",
-        //         shipments = shipments.Select(shipment => Shipment.Create(shipment)).ToList(),
-        //         carrier_accounts = new List<CarrierAccount>() { new CarrierAccount() { id = "ca_7642d249fdcf47bcb5da9ea34c96dfcf" } }
-        //     };
-
-        //     order.Create();
-
-        //     Assert.IsNotNull(order.id);
-        //     Assert.AreEqual(order.reference, "OrderRef");
-        //     CollectionAssert.AreEqual(
-        //         new List<string>() { "ca_7642d249fdcf47bcb5da9ea34c96dfcf" },
-        //         new HashSet<string>(order.shipments.SelectMany(s => s.rates).Select(r => r.carrier_account_id)).ToList()
-        //     );
-        // }
-
-        [TestMethod]
-        public void TestBuyOrder()
+        private static Order CreateBasicOrder()
         {
-            Order order = Order.Create(parameters);
-            order.Buy("USPS", "Priority");
-
-            Assert.IsNotNull(order.shipments[0].postage_label);
+            return Order.Create(Fixture.BasicOrder);
         }
 
         [TestMethod]
-        public void TestCreateAndRetrieveOrder()
+        public void TestCreate()
         {
-            Order order = Order.Create(parameters);
+            VCR.Replay("create");
 
-            Assert.IsNotNull(order.id);
-            Assert.AreEqual(order.reference, "OrderRef");
+            Order order = CreateBasicOrder();
 
-            Order retrieved = Order.Retrieve(order.id);
-            Assert.AreEqual(order.id, retrieved.id);
+            Assert.IsInstanceOfType(order, typeof(Order));
+            Assert.IsTrue(order.id.StartsWith("order_"));
+            Assert.IsNotNull(order.rates);
         }
 
         [TestMethod]
-        [ExpectedException(typeof(ResourceAlreadyCreated))]
-        public void TestCreateOrderWithId()
+        public void TestRetrieve()
         {
-            Order order = new Order
-            {
-                id = "order_asjhd"
-            };
-            order.Create();
-        }
+            VCR.Replay("retrieve");
 
-        [TestMethod]
-        [ExpectedException(typeof(HttpException))]
-        public void TestFailure() => Order.Create(new Dictionary<string, object>());
+            Order order = CreateBasicOrder();
+
+            // use "TestCreate"
+            Order retrievedOrder = Order.Retrieve(order.id);
+
+            Assert.IsInstanceOfType(retrievedOrder, typeof(Order));
+            Assert.AreEqual(order.id, retrievedOrder.id);
+        }
 
         [TestMethod]
         public void TestGetRates()
         {
-            Order order = Order.Create(parameters);
-            List<Rate> old = order.rates;
+            VCR.Replay("get_rates");
+
+            // use "TestCreate"
+            Order order = CreateBasicOrder();
+
+            // TODO: edit after get_rates PR
```

Comment:
What's this TODO for? This is valid syntax that isn't related to the removal of `GetRates` from the shipment object. You cannot regenerate rates for an order, only retrieve them and as such, this is staying.

Reply:
Resolved in 88726861fba93fd348a76b99a851ecbdaf1c49d5